### PR TITLE
Add some configurations to readme

### PR DESCRIPTION
**Pull request by JoelPasvolsky**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -46,6 +46,56 @@ corner lets you edit the following information:
...
+
+Configuration
+-------------
+
+Some more advanced configurations include the following:
+
+Repository Requirements
+~~~~~~~~~~~~~~~~~~~~~~~
+
+You can create a YAML file to open a repository with your required packages
```
Comment (randomir) [inline README.rst:61]: [a YAML file] named `.gitpod.yml` and placed in the repo root [...]

Comment (randomir) [reply README.rst:61]: Also, good place to link to [the gp docs](https://www.gitpod.io/docs/config-gitpod-file/).

File: README.rst [hunk 2]
```diff
@@ -46,6 +46,56 @@ corner lets you edit the following information:
...
+
+  tasks:
+   - init: pip install -r requirements.txt
+
+User Dockerfile
+~~~~~~~~~~~~~~~
+
+The IDE uses a Docker image, based on python:3.7-slim, available within
+the IDE as ``dwavesys/leapide:latest``. To build on top of it, your Dockerfile
+should look like this::
```
Comment (randomir) [inline README.rst:72]: Also, the dockerfile has to be [referenced from the `.gitpod.yml`](https://www.gitpod.io/docs/config-docker/).

File: README.rst [hunk 3]
```diff
@@ -46,6 +46,56 @@ corner lets you edit the following information:
...
+          clang \
+          libboost-dev \
+      && apt-get clean && rm -rf /var/lib/apt/lists/* /tmp/*
+
+  # install system-wide python package
+  RUN pip install --no-cache-dir \
+          dwave-ocean-sdk==4.0.0 \
+      && rm -rf /tmp/*
+
+API token
```
Comment (randomir) [inline README.rst:90]: Maybe mention that this is valid for new users only (registered after Dec 2).

Old users can get this feature by 1) deleting `DWAVE_API_TOKEN` env var and 2) creating a new workspace.

Comment (JoelPasvolsky) [reply README.rst:90]: I'm thinking this info is more relevant to release notes. If we thought large numbers of users might be affected that would justify transition info in the tool's documentation but I don't believe that's the case.

**Pull request merged**